### PR TITLE
Features

### PR DESCRIPTION
**Pull request by Thermiee**
## Description
This React capstone project is about building a mobile web application to check a list of Animes using React and Redux and
 Api from`https://chandan-02.github.io/anime-facts-rest-api/` 

### General requirements

- [x] There are no linter errors.
- [x] Followed the **Github flow**.
- [x] The documentation is professional.

### Milestone requirements:

- [x] Use React documentation.
- [x] Use React components.
- [x] Use React props.
- [x] Use React Router.
- [x] Connect React and Redux.
- [x] Handle events in a React app.
- [x] Write integration tests with a React testing library.
- [x] Use styles in a React app.
- [x] Use React life cycle methods.
- [x] Apply React best practices and language style guides in code.
- [x] Use store, actions, and reducers in React.

Thank you for taking the time to review this pull request. :pray:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,67 @@
...
 Feel free to check the [issues page](../../issues/).
 
-## Show your support
+## Acknowledgments
 
-Give a ⭐️ if you like this project!
+Thank you Microverse for the opportunity!
 
-## Acknowledgments
+## Show your support
```
Comment (MrBrN197) [inline README.md]: - [x] Please ensure to give credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 2]
```diff
@@ -1,73 +1,67 @@
...
+`npm install`
 
-### Usage
+Run the live server
 
-### Run tests
+`npm run start`
 
-### Deployment
+Go to [Localhost](http://localhost:3000)
```
Comment (MrBrN197) [inline README.md]: - [x] as part of the project you are also required to deploy this website. please ensure you deploy this project to either GH pages, Netlify or Heroku.

File: src/App.test.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+test('renders learn react link', () => {
+  const { getByText } = render(
+    <Provider store={store}>
+      <App />
+    </Provider>,
+  );
+
+  expect(getByText(/learn/i)).toBeInTheDocument();
+});
```
Comment (MrBrN197) [inline src/App.test.js]: - [x] Hi, please note that you should have unit & integration(using React Testing Library) tests for most of your components and some of your pure functions in the Redux code.

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @Thermiee  👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!
## To highlight 👏👏
- ✔ linter checks all passing
- ✔ page functionality is working well

## Required Changes ♻️
- [x] Hello well done on the project. I'd like you to note that you need to have a video demo showcasing your project with a clear description of it including a particularly interesting piece of functionality you'd like to highlight. 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1,73 +1,73 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Anime-World
 
-> Description the project.
+> "Anime-World" is a website show list of animes and show different facts about them.
 
+[Live Demo](scintillating-khapse-6ebe08.netlify.app)
```
Comment (rotimiazeez) [inline README.md]: - [ ] Please do provide a valid deploy link to your project, the current link is broken🤔

File: src/App.test.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  ...render(<Provider store={store}>{ui}</Provider>),
+  store,
+});
+
+describe('App', () => {
+  it('renders without crashing', () => {
+    const { getByText } = renderWithRedux(<App />, { store });
+    const linkElement = getByText(/2022 AnimeWorld/i);
+    expect(linkElement).toBeInTheDocument();
+  });
```
Comment (rotimiazeez) [inline src/App.test.js]: - [ ] Kindly make sure you have unit tests for several components of the app and some pure functions in Redux code.
- [ ] You have integration tests using React Testing Library as suggested by the previous reviewer💪🏽

File: README.md [hunk 6]
```diff
@@ -1,73 +1,73 @@
...
 
-## Built With
+[Video Demo](https://www.loom.com/share/8302a8a49d7a493388745ddbfdaf65bf)
```
Comment (rotimiazeez) [inline README.md]: - [ ] Kindly make sure your presentation video contains a highlight about a piece of the code the student finds interesting or is proud of making not only the page itself.

File: README.md (hunk 2 above)
Comment (rotimiazeez) [reply README.md]: - [ ] I'd suggest you either remove this or add your working deploy link here🤷‍♂️

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi <STUDENT_MENTION>, 

### Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bellom** [comment]
@Thermiee - Good job, however, you're missing the command to run your test in the README. kindly add the command.

**Pull request merged**